dhcpv4/dhcpv4.go

```diff
@@ -222,8 +222,7 @@ func PrependModifiers(m []Modifier, other ...Modifier) []Modifier {
 // NewInform builds a new DHCPv4 Informational message with the specified
 // hardware address.
 func NewInform(hwaddr net.HardwareAddr, localIP net.IP, modifiers ...Modifier) (*DHCPv4, error) {
-	return New(PrependModifiers(
-		modifiers,
+	return New(PrependModifiers(modifiers,
```

Comment:
Why is this changing?

Reply:
As mentioned in the updated description, to make it consistent with the neighboring functions such as `NewDiscovery`, `NewRequestFromOffer`, `NewRenewFromOffer` etc. All of them start with `return New(PrependModifiers(modifiers,`.

Reply:
I mean, nothing is changing in here other than removing a line. Is it needed? Can it be avoided to make this PR more clear?

Reply:
I can avoid it if desired, but in my opinion the tradeoff of making this PR slightly less clear in order to ultimately make the code clearer for all future contributors and maintainers is worth it. I felt like this change is topical for this PR instead of a separate one as I'm working on cleaning up `NewInform`, but in this end it is still your call if we keep it or remove it.

Reply:
Separated: https://github.com/insomniacslk/dhcp/pull/481